pandas/tests/frame/indexing/test_setitem.py

```diff
@@ -379,6 +379,15 @@ def test_setitem_complete_column_with_array(self):
         assert expected["d"].dtype == arr.dtype
         tm.assert_frame_equal(df, expected)
 
+    def test_setitem_periodd_dtype(self):
```

Comment:
typo extra "d"

Reply:
> typo extra "d"

Originally its `period[D]` but to name a function I thought this would be fine.

But I think there are ways we can rename
1. `..._period_d_dtpye(self)`
2. `..._period_dtype(self)`

What do you think?

Reply:
`_period_d_dtype` is fine

Reply:
Okay! sounds good 👍
Thanks